service/src/test/java/bio/terra/pipelines/dependencies/leonardo/LeonardoServiceTest.java

```diff
@@ -0,0 +1,113 @@
+package bio.terra.pipelines.dependencies.leonardo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+import bio.terra.common.iam.BearerToken;
+import bio.terra.pipelines.app.configuration.internal.ImputationConfiguration;
+import bio.terra.pipelines.app.configuration.internal.RetryConfiguration;
+import java.net.SocketTimeoutException;
+import java.util.List;
+import java.util.UUID;
+import org.broadinstitute.dsde.workbench.client.leonardo.ApiException;
+import org.broadinstitute.dsde.workbench.client.leonardo.api.AppsApi;
+import org.broadinstitute.dsde.workbench.client.leonardo.model.ListAppResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.stubbing.Answer;
+import org.springframework.retry.backoff.FixedBackOffPolicy;
+import org.springframework.retry.support.RetryTemplate;
+
+@ExtendWith(MockitoExtension.class)
+class LeonardoServiceTest {
+  final String workspaceId = UUID.randomUUID().toString();
+  final ImputationConfiguration wdsServerConfiguration = new ImputationConfiguration(workspaceId);
```

Comment:
is this variable name `wdsServerConfiguration` a copy-pasta remnant?

Reply:
whoops 😬